### DISCUSSION TITLE
无法在ubuntu ecs和docker中 实现安装

### DISCUSSION THREAD

**Discussion by ChunelFeng** [xmake-io/xmake#2081] [Q&A]
ubuntu中无法使用以下命令完成xmake安装，

自己在mac上创建的ubuntu的docker环境也无法执行成功

请求帮助

<img width="577" alt="image" src="https://user-images.githubusercontent.com/37905059/154854848-2adccde8-c906-4501-8dcc-e242c24066c3.png">

<img width="581" alt="image" src="https://user-images.githubusercontent.com/37905059/154854724-3738b1f5-fae8-4b7a-aadc-a1f36244426e.png">

**Comment by ChunelFeng** [comment]
安装图片中描述操作，也无法完成功能

**Comment by waruqi** [comment]
> 安装图片中描述操作，也无法完成功能

请给出完整执行命令和输出

另外，不建议走 apt 安装，官方和github 首页 不都有一键安装命令么，干嘛不用
